product_docs/docs/pgd/5/appusage/extensions.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Using other extensions with PGD
```

Comment:
```suggestion
title: Using extensions with PGD
```

Readers are unlikely to think of PGD (naturally) as an extension itself on first read, so this should make it more obvious.

Reply:
Good point.